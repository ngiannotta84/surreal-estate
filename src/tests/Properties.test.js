import React from "react";
import { render, screen } from "@testing-library/react";
import Properties from "../components/Properties";

describe("Properties", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Properties />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("renders correct text", () => {
    render(<Properties />);
    const text = screen.getByText(/properties page/i);
    expect(text).toBeInTheDocument();
  });
});
