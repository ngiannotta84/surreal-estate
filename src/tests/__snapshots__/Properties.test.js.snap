// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Properties renders correctly 1`] = `
<DocumentFragment>
  <div>
    Properties page
  </div>
</DocumentFragment>
`;
